### ISSUE TITLE
Plantuml server does not support https

### ISSUE THREAD

**Issue by raphael-leger** [eugene-manuilov/typeorm-uml#30]
Hi again,

The default plantuml url should probably be `http://www.plantuml.com/plantuml` (http, not https)

https://github.com/eugene-manuilov/typeorm-uml/blob/cf123d5debcea30dd1122ee7f44c8985245b924e/src/builder/typeorm-uml.class.ts#L95

With the current default, executing `new TypeormUml().build(...)` produces the following error:
```
(node:12798) UnhandledPromiseRejectionWarning: TypeError [ERR_INVALID_PROTOCOL]: Protocol "https:" not supported. Expected "http:"
    at new ClientRequest (_http_client.js:155:11)
    at request (http.js:50:10)
    at Object.get (http.js:54:15)
    at /myproject/node_modules/typeorm-uml/lib/builder/typeorm-uml.class.js:63:20
```

To be noted that when accessing http://www.plantuml.com/plantuml or https://www.plantuml.com/plantuml in a browser (chrome at least), it shows up 'Not Secure' in the status bar.

**Comment by eugene-manuilov** [comment]
Thanks, @raphael-leger! Updated and published as `@next` for now.

**Comment by eugene-manuilov** [comment]
Ok, it has been shipped with the new version `1.6.0`. Closing this ticket.
